api/openapi/users.yml

```diff
@@ -305,15 +362,20 @@ paths:
           $ref: "#/components/responses/UserRes"
         "400":
           description: Failed due to malformed JSON.
-        "404":
-          description: Failed due to non existing user.
         "401":
           description: Missing or invalid access token provided.
+        "404":
+          description: Failed due to non existing user.
+        "415":
+          description: Missing or invalid content type.
+        "422":
+          description: Database can't process request.
         "500":
           $ref: "#/components/responses/ServiceError"
 
   /password/reset-request:
     post:
+      operationId: requestPasswordReset
```

Comment:
`/password` is comes under the `/users` route 

password reset request end point 
`/users/passsword/reset-request`
`/users/password/reset`